staging/src/k8s.io/client-go/scale/fake/client.go

```diff
@@ -60,7 +60,7 @@ func (f *fakeNamespacedScaleClient) Get(ctx context.Context, resource schema.Gro
 
 func (f *fakeNamespacedScaleClient) Update(ctx context.Context, resource schema.GroupResource, scale *autoscalingapi.Scale, opts metav1.UpdateOptions) (*autoscalingapi.Scale, error) {
 	obj, err := f.fake.
-		Invokes(testing.NewUpdateSubresourceAction(resource.WithVersion(""), f.namespace, "scale", scale), &autoscalingapi.Scale{})
+		Invokes(testing.NewUpdateSubresourceAction(resource.WithVersion(""), "scale", f.namespace, scale), &autoscalingapi.Scale{})
```

Comment:
I went to build some tests using this fake client and was scratching my head as to why the Reactor I was building wasn't behaving as I expected. It turns out that this was previously swapping the namespace and subresource parameters. 

Oddly enough, it doesn't appear to cause any major functionality issues, but if you are trying to do any validation or logic depending on either the Subresource or Namespace fields of the Action you are in for a surprise because they are swapped.